vultr/client.go

```diff
@@ -0,0 +1,105 @@
+package vultr
+
+import (
+	"context"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/vultr/govultr"
+	"github.com/libdns/libdns"
+)
+
+type Client struct {
```

Comment:
This could probably be un-exported, right?

Reply:
I think so, it's only used internally by the package. I will test if it works by unexporting this variable.